wordpress-mobile/release-toolkit
# Add Upload to S3 Action

Adds an "upload to S3" action inside the release toolkit that's useful for binary storage (particularly for installable builds).

**To Test:**
Note that https://github.com/woocommerce/woocommerce-android/pull/5862 passes and properly displays a comment with a download link – the upload functionality for this is powered by this new action.
